gapis/api/vulkan/api/properties_features_requirements.api

```diff
@@ -487,6 +487,7 @@ sub void GetPhysicalDeviceProperties2(
       next := MutableVoidPtr(as!void*(props.pNext))
       for i in (0 .. numPNext) {
         sType := as!const VkStructureType*(next.Ptr)[0:1][0]
```

Comment:
You should't need another _ = sType, the read should be handled by 
as!const VkStructureType*(next.Ptr)[0:1]**[0]**